index.js

```diff
@@ -11,7 +11,7 @@ var VirtualSerialPort = function(path, options, openImmediately, callback) {
     };
 
     if(openImmediately || openImmediately === undefined || openImmediately === null) {
-        process.nextTick(function() {
+        process.nextTick(() => {
```

Comment:
I believe this library still works with node 0.10 (which I personally require at the moment). However this change requires at least 4. You should probably not introduce a dependency on a new node version in a small bugfix like this.